terraform/azure_server/main.tf

```diff
@@ -5,13 +5,12 @@ module "images" {
 locals {
   scripts = var.dev_tools && module.images.source_images[var.image].os == "windows" ? concat(
     [
-      file("${path.module}/files/enable_containers.ps1"),
-      file("${path.module}/files/install-choco.ps1"),
-      # need to run it twice since first time will fail when it installs .NET Framework
-      file("${path.module}/files/install-choco.ps1"),
-      file("${path.module}/files/setup-choco.ps1"),
-      file("${path.module}/files/install-tools.ps1"),
-      file("${path.module}/files/install-docker.ps1"),
+      file("${path.module}/files/enable_features.ps1"),
```

Comment:
enable_features now contains the initial choco install that gets .NET Framework installed, since we just do all operations that need to happen before a restart once instead of doing the restart twice.